src/main/java/com/zendesk/maxwell/BufferedMaxwell.java

```diff
@@ -0,0 +1,23 @@
+package com.zendesk.maxwell;
+
+import com.zendesk.maxwell.producer.BufferedProducer;
+
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.concurrent.TimeUnit;
+
+/**
+ * Created by ben on 8/27/16.
+ */
+public class BufferedMaxwell extends Maxwell {
+	public BufferedMaxwell(MaxwellConfig config) throws SQLException {
+		super(config);
+		config.producerType = "buffer";
+	}
+
+	public RowMap getRow(long timeout, TimeUnit unit) throws IOException, InterruptedException {
```

Comment:
totally nitpick, the name is a bit weird. At first I thought about MaxwellReplicator#getRow... getProducedMessage/getSentRecord/pollProducerRecords/whatever name that makes sense in english =)